### ISSUE TITLE
Ethernet cable drops connection.

### ISSUE THREAD

**Issue by ahmadgazar** [robotology/icub-support#677]
### Description of the failure
Ethernet cable of the robot keeps dropping connection from time to time.

### Detailed conditions and logs of the failure
The connection with `icub-head` was lost from time to time, to check whether this is a problem of the cable of the robot, we used an external cable to connect the robot to the local network. So far the connection has been stable with the external cable, so we think there is a problem with the cable of the robot.

**Comment by julijenv** [comment]
hi @ahmadgazar ,
where is the log??!?!!?!

**Comment by hu-yue** [comment]
@julijenv we don't have the log because nothing is shown in the log as the connection just drops. Unlike in #675, we don't have access to the `icub-head`.
We took the extension cable off and attached the original cable directly to the power supply and the connections seems stable. So it might be that there is a problem with the extension cable.

**Comment by GiulioRomualdi** [comment]
@julijenv actually the ethernet drop just happened again without the extension (after a couple of hours), so maybe the problem is not the extension. The network is fine because we detached the cable and the robot automatically connected to the wifi.
At this point, maybe it's the cable of the robot, once (about one month ago) we had to change the ethernet cable connector because it was having bad contacts, maybe it is the same or similar issue.

**Comment by hu-yue** [comment]
@julijenv during the tests for the demo yesterday (Tuesday), with @GiulioRomualdi and @ahmadgazar we realized that the problem is most probably not the cable, as even with external cables (we tried two differently ones), the connection was lost sometimes. We then performed the demo with the WiFi which seemed to guarantee a more stable connection. So maybe the connector on the robot side is broken or something similar?
